core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -425,13 +431,18 @@ public LockedInodePath lockInodePath(AlluxioURI uri, LockPattern lockPattern)
    * @param uri the uri to lock
    * @param lockPattern the {@link LockPattern} to lock the inodes with
    * @param tryLock true to use {@link Lock#tryLock()} or false to use {@link Lock#lock()}
+   * @param journalContext the journal context to flush when the LockedInodePath releases
    * @return the {@link LockedInodePath} representing the locked path of inodes
    * @throws InvalidPathException if the path is invalid
    */
-  public LockedInodePath lockInodePath(AlluxioURI uri, LockPattern lockPattern, boolean tryLock)
+  public LockedInodePath lockInodePath(
+      AlluxioURI uri, LockPattern lockPattern, boolean tryLock, JournalContext journalContext
+  )
```

Comment:
nit: we don't often see this indentation pattern in the codebase

Reply:
what should i change to though...?